tests/cases/eth2eth/tests/contract_test.go

```diff
@@ -0,0 +1,167 @@
+package tests
```

Comment:
I don't think it is necessary since it has already been tested with yui-ibc-solidity.
Is there any difference between the test in yui-ibc-solidity and the test in yui ibc-solidity?

Reply:
Certainly the same thing we're doing here.
https://github.com/hyperledger-labs/yui-ibc-solidity/blob/main/tests/integration/contract_test.go
@siburu What do you think?

Reply:
No, there's no need to do integration testing here.

Instead you need to e2e testing for ICS-20 like below.
https://github.com/datachainlab/yui-relayer-build/blob/v0.3/tests/cases/tm2tm/scripts/test-tx

Reply:
To do so, you need to write a CLI tool for kicking `sendTransfer`.

Reply:
https://github.com/datachainlab/yui-relayer-build/pull/5/commits/be9bc26a99d5e9de35f7d73ce99d4289a2b91aab
Added cli command to `/tests/chains/ethereum/scripts/cmd/*`